frontend/class-twitter.php

```diff
@@ -20,135 +20,34 @@ class WPSEO_Twitter {
 	public static $instance;
 
 	/**
-	 * Images.
-	 *
-	 * @var array
-	 */
-	private $images = array();
-
-	/**
-	 * Images.
-	 *
-	 * @var array
-	 */
-	public $shown_images = array();
```

Comment:
I would keep this property because it is public.

Reply:
Done